Project1/MapTest.cpp

```diff
@@ -15,10 +15,12 @@ bool test::runMapTest() {
 	FairDice<100> d;
 	UnitData mia_data = { "Mia", d, AttributeList({ 5, 0, 8, 10, 19, 4, 6, 1 }) };
 	UnitData ike_data = { "Ike", d, AttributeList({ 5, 0, 8, 10, 19, 4, 6, 1 }) };
+	UnitPtr mia = std::make_unique<Unit>(mia_data);
+	UnitPtr ike = std::make_unique<Unit>(ike_data);
 	auto unit_vec = std::vector<UnitData>();
 	unit_vec.push_back(mia_data);
 	unit_vec.push_back(ike_data);
-	PartyData data1 = { "Greil Mercenaries", unit_vec};
+	PartyData data1 = { "Greil Mercenaries"};
 	PartyData data2 = { "Daein" };
 	std::vector<PartyData> party_vec = std::vector<PartyData>();
```

Comment:
No change: just fyi you can init vector like `std::vector<PartyData> party_vec = { data1, data2 };'`

Reply:
I know, but when I'm testing stuff I modify the lines a bunch, and it easier when everything is on its own line.